### PR TITLE
[cypress] logged in user test case

### PR DESCRIPTION
**Pull request by prachiyadav**
SMOKE: Log in to kiali and check if correct user is logged in - top right
- https://issues.redhat.com/browse/OSSM-1172

### CODE REVIEW

File: frontend/cypress/integration/common/kiali_user.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        cy.get('button[type="submit"]').click()
+    }
+})
+
+And('user see console in URL', () => {
+    cy.url().should('include', 'console')
+})
+
+Then('user locate User in UI and verify' ,() => {
+    cy.get('.pf-c-dropdown__toggle-text').contains('jenkins')
```
Comment (ScriptingShrimp) [inline frontend/cypress/integration/common/kiali_user.ts:54]: `contains('jenkins')` please change hard-coded value to a variable defined at top `USERNAME` 

`const USERNAME = Cypress.env('USERNAME') || 'jenkins';` should be defined in this file

File: frontend/cypress/integration/featureFiles/kiali_user.feature [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Feature: Kiali user
+
+  User wants to verify login user
+
+  Scenario: locate user on kiali page and verify
+    Given user opens base url
+    And user clicks Log In With OpenShift
+    And user clicks my_htpasswd_provider
+    And user fill in username and password
+    And user see console in URL
```
Comment (ScriptingShrimp) [inline frontend/cypress/integration/featureFiles/kiali_user.feature:10]: please refactor step `Given user opens base url` with new step `Given user is at administrator perspective` 
for correct usage see
https://github.com/kiali/kiali/blob/bd662370f0950963d1151825915fc9e91db39474/frontend/cypress/integration/featureFiles/kiali_cookie.feature#L6

File: frontend/cypress/integration/common/kiali_user.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    if (auth_strategy === 'openshift') {
+        cy.get('#inputUsername').type('' || USERNAME);
+        cy.get('#inputPassword').type('' || PASSWD);
+        cy.get('button[type="submit"]').click()
+    }
+})
+
+And('user see console in URL', () => {
+    cy.url().should('include', 'console')
+})
```
Comment (ScriptingShrimp) [inline frontend/cypress/integration/common/kiali_user.ts:51]: After you refactor feature file, you can delete this steps. Testcase should work the same we as it did before

File: frontend/cypress/integration/common/kiali_user.ts (hunk 1 above)
Comment (ScriptingShrimp) [reply frontend/cypress/integration/common/kiali_user.ts:54]: please validate that this testcase is working with `kubeadmin` username

**Review by ScriptingShrimp — CHANGES_REQUESTED** [review]
please address required changes, rebase and squash commits. +1

**Comment by lucasponce** [comment]
What's the status of this PR?

**Comment by lucasponce** [comment]
Closing. If there is activity on this PR, please re-open it. I'll close PRs within 2 months without activity.

**Pull request closed without merging**